api/db-cdr/generate-cdr/make-bq-data.sh

```diff
@@ -179,7 +179,7 @@ bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
 (concept_id, concept_name, domain_id, vocabulary_id, concept_class_id, standard_concept,
 concept_code, count_value, prevalence, source_count_value, synonyms)
 select c.concept_id, c.concept_name, c.domain_id, c.vocabulary_id, c.concept_class_id, c.standard_concept, c.concept_code,
-0 as count_value , 0.0 as prevalence, 0 as source_count_value,concat(cast(c.concept_id as string),'\\|',string_agg(cs.concept_synonym_name,'\\|')) as synonyms
+0 as count_value , 0.0 as prevalence, 0 as source_count_value,concat(cast(c.concept_id as string),'|',string_agg(replace(cs.concept_synonym_name,'|','||'),'|')) as synonyms
```

Comment:
This string_agg should be \| instead of ||

Reply:
string_agg is '|'. It is the replace that is replacing any | in concept_synonym_name with ||.

Reply:
The string_agg is using |... it's just replacing | with || in the synonyms before doing that. Which sounds correct.

Reply:
So then you would just split on '|' but not '||' ?

Reply:
You'd need to have parsing logic which splits on '|' but replaces '||' with '|' to unescape things. Haven't quite worked out what the right code for this will be, but there's a way. :)